lib/beaker/module_install_helper.rb

```diff
@@ -144,6 +167,22 @@ def get_module_source_directory(search_in)
     end
     module_source_dir
   end
+
+  def forge_host
+    fh = ENV['BEAKER_FORGE_HOST'] || ENV['forge_host']
```

Comment:
Where is the `|| ENV['forge_host']` coming from? It is not described in the README.